prow/prowjobs/GoogleCloudPlatform/gcp-guest/gcp-guest-config.yaml

```diff
@@ -409,4 +409,23 @@ periodics:
       - "-zone=us-west1-a"
       - "-images=projects/bct-prod-images/global/images/family/debian-10,projects/bct-prod-images/global/images/family/debian-11,projects/bct-prod-images/global/images/family/centos-7,projects/bct-prod-images/global/images/family/centos-stream-8,projects/bct-prod-images/global/images/family/centos-stream-9,projects/bct-prod-images/global/images/family/rhel-7,projects/bct-prod-images/global/images/family/rhel-8,projects/bct-prod-images/global/images/family/rhel-9,projects/bct-prod-images/global/images/family/rocky-linux-8,projects/bct-prod-images/global/images/family/rocky-linux-8-optimized-gcp,projects/bct-prod-images/global/images/family/almalinux-8"
       - "-test_projects=compute-image-test-pool-002,compute-image-test-pool-003,compute-image-test-pool-004,compute-image-test-pool-005"
-      - "-exclude=oslogin"
+      - "-exclude=(oslogin)|(networkperf)|(storageperf)"
```

Comment:
isn't there other builds that will requirer excluding networkpef and storageperf?

Reply:
Zach is out today so I can't confirm, however, 

I believe the tests "storageperf" and "networkperf" are a special category of tests. They should not block releases if they fail, but we want them on the test grid to show other teams that the network and disk performance is good.

This extra test grid, cit-periodics-performance, is what I'm looking to add for that second requirement of showing other teams the statistics.